fix_failing_tests.py

```
@@ -0,0 +1,133 @@
+    ]
+
+    for pattern, replacement in patterns_to_replace:
+        content = re.sub(pattern, replacement, content)
+
+    try:
+        with open(file_path, "w") as f:
+            _ = f.write(content)
+    except PermissionError:
+        print(f"❌ Ошибка: Нет доступа для записи в файл {file_path}", file=sys.stderr)
+        return False
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)